docs/branching.md

```diff
@@ -0,0 +1,9 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+   -
+   - SPDX-License-Identifier: CC0-1.0
```

Comment:
Do we _want_ to keep Serokell copyrights here (and also in a bunch of other places)

Reply:
There are a few files, including this one, that are exact copies from elsewhere (mostly [metatemplates](https://github.com/serokell/metatemplates)), that aren't expected to change and shouldn't really be copywritable (see [our FAQ](https://www.notion.so/serokell/FAQ-8da3477b8b7c4faaab6460ff38fc8929) on the matter).

So, I checked again and changed some of them that were using our `CC0-1.0`, but I left the ones that fall under the category above.